c++/test/TestColumnReaderInvalidStripes.cc

```diff
@@ -0,0 +1,267 @@
+
+#include "Adaptor.hh"
+#include "ColumnReader.hh"
+#include "orc/Exceptions.hh"
+#include "OrcTest.hh"
+
+#include "wrap/orc-proto-wrapper.hh"
+#include "wrap/gtest-wrapper.h"
+#include "wrap/gmock.h"
+
+#include <cmath>
+#include <iostream>
+#include <vector>
+
+#ifdef __clang__
+  DIAGNOSTIC_IGNORE("-Winconsistent-missing-override")
+  DIAGNOSTIC_IGNORE("-Wmissing-variable-declarations")
+#endif
+#ifdef __GNUC__
+  DIAGNOSTIC_IGNORE("-Wparentheses")
+#endif
+
+namespace orc {
+  using ::testing::TestWithParam;
+  using ::testing::Values;
+
+  class TestColumnReaderInvalidStripes : public ::testing::Test {
+    class MockStripeStreams : public StripeStreams {
+    public:
+      virtual ~MockStripeStreams() override;
+
+      std::unique_ptr<SeekableInputStream> getStream(uint64_t columnId,
+                                                    proto::Stream_Kind kind,
+                                                    bool stream)
+                                                    const override {
+      return std::unique_ptr<SeekableInputStream>
+              (getStreamProxy(columnId, kind, stream));
+    }
```

Comment:
Fix alignment and style, same for below lines.

Reply:
Is there any problem with the MOCK_CONST_METHODN styles? I found the alignment in TestColumnReader.cc pretty ugly.